reducer/src/main/java/com/graphicsfuzz/reducer/ReductionDriver.java

```diff
@@ -450,10 +467,20 @@ private void writeState(ShaderJob state, File shaderJobFileOutput,
     if (addInitializers) {
       AddInitializers.addInitializers(stateToWrite);
     }
-    fileOps.writeShaderJobFile(
-        stateToWrite,
-        shaderJobFileOutput
-    );
+
+    if (literalToUniformPassManager.isPresent()) {
+      fileOps.writeShaderJobFile(
+          stateToWrite,
+          shaderJobFileOutput,
+          Optional.of(new PipelineUniformValueSupplier(stateToWrite.getPipelineInfo()))
+      );
```

Comment:
Let's always do this, regardless of whether we have a "literal to uniform" pass manager.

It's always a nice idea to emit comments showing the known values of uniforms - it can only make the final shader more readable, so I figure we should do it regardless of the pass manager.

Reply:
i.e., revert this change to what was there before, but pass the extra "Optional.of(...)" argument.